### PR TITLE
Update compute-capacity-limits-by-edition-of-sql-server.md

### PR DESCRIPTION
**Pull request by PhilB-MSFT**
Update with note on 64 core per NUMA node limit - see IcM 423726918.

### CODE REVIEW

**Comment by Court72** [comment]
@MikeRayMSFT

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/sql-server/compute-capacity-limits-by-edition-of-sql-server.md [hunk 1]
```diff
@@ -83,6 +83,9 @@ The following table specifies the compute capacity limits for a single instance
 
 <sup>1</sup> Enterprise Edition with Server + Client Access License (CAL) licensing is limited to 20 cores per [!INCLUDE [ssNoVersion](../includes/ssnoversion-md.md)] instance. (This licensing isn't available for new agreements.) There are no limits under the Core-based Server Licensing model.
 
+> [!NOTE]  
+> SQL Server limit of 64 logical processors per NUMA node.
```
Comment (rwestMSFT) [inline docs/sql-server/compute-capacity-limits-by-edition-of-sql-server.md]:
```suggestion
> SQL Server limits the number of logical processors per NUMA node to 64.
```

File: docs/sql-server/compute-capacity-limits-by-edition-of-sql-server.md [hunk 2]
```diff
@@ -83,6 +83,9 @@ The following table specifies the compute capacity limits for a single instance
 
 <sup>1</sup> Enterprise Edition with Server + Client Access License (CAL) licensing is limited to 20 cores per [!INCLUDE [ssNoVersion](../includes/ssnoversion-md.md)] instance. (This licensing isn't available for new agreements.) There are no limits under the Core-based Server Licensing model.
 
+> [!NOTE]  
+> SQL Server limits the number of logical processors per NUMA node to 64.
```
Comment (rwestMSFT) [inline docs/sql-server/compute-capacity-limits-by-edition-of-sql-server.md]:
```suggestion
> [!INCLUDE [ssNoVersion](../includes/ssnoversion-md.md)] limits the number of logical processors per NUMA node to 64.
```

**Comment by rwestMSFT** [comment]
@PhilB-MSFT Do you mind making changes like this to `sql-docs-pr` in future? Thanks!

**Comment by PhilB-MSFT** [comment]
> @PhilB-MSFT Do you mind making changes like this to `sql-docs-pr` in future? Thanks!

My apologies, I'll remember to use that in the future.

**Pull request merged**